Casks/f/freeplane.rb

```diff
@@ -13,8 +13,7 @@
 
   livecheck do
     url "https://sourceforge.net/projects/freeplane/rss?path=/freeplane%20stable"
-    regex(%r{/freeplane%20stable/Freeplane[._-]v?(\d+(?:\.\d+)+)(?:[._-]#{arch})?\.dmg}i)
-    strategy :page_match
+    regex(/Freeplane[._-]v?(\d+(?:\.\d+)+)(?:[._-]#{arch})?\.dmg/i)
```

Comment:
```suggestion
    regex(%r{url=.*?/Freeplane[._-]v?(\d+(?:\.\d+)+)(?:[._-]#{arch})?\.dmg}i)
```

Just some tweaks to bring this in line with typical regex patterns for `Sourceforge` `livecheck` blocks.